ci-operator/step-registry/baremetal/lab/post/firewall/baremetal-lab-post-firewall-commands.sh

```diff
@@ -33,14 +33,27 @@ for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
   IP_ARRAY+=( "$ip" )
 done
 
+if [[ -f "${SHARED_DIR}/ipi_bootstrap_ip_address" ]]; then
+  BOOTSTRAP_IP="$(<"${SHARED_DIR}/ipi_bootstrap_ip_address")"
+fi
+
 echo 'Deprovisioning firewall configuration'
 timeout -s 9 10m ssh "${SSHOPTS[@]}" "root@${AUX_HOST}" bash -s -- \
-  "${INTERNAL_NET_CIDR}" "${IP_ARRAY[@]}" << 'EOF'
+  "${INTERNAL_NET_CIDR}" "${IP_ARRAY[@]}" "${BOOTSTRAP_IP}" << 'EOF'
   set -o nounset
   set -o errexit
   INTERNAL_NET_CIDR="${1}"
   IP_ARRAY="${@:2}"
+  BOOTSTRAP_IP="${3}"
   for ip in $IP_ARRAY; do
+    # TODO: change to firewalld or nftables
     iptables -D FORWARD -s ${ip} ! -d "${INTERNAL_NET_CIDR}" ! -p tcp --dport 22 -j DROP
+    iptables -D FORWARD -s ${ip} -d 192.168.70.0/24 -j ACCEPT
   done
-EOF
\ No newline at end of file
+  if [ -n "${BOOTSTRAP_IP}" ]; then
+    RULE=$(sed 's/^-A /-D /' <(iptables -S FORWARD | grep "${BOOTSTRAP_IP}"| grep DROP))
+    [[ $RULE =~ D.*$BOOTSTRAP_IP.*DROP ]] && iptables ${RULE}
+    RULE=$(sed 's/^-A /-D /' <(iptables -S FORWARD | grep "${BOOTSTRAP_IP}"| grep ACCEPT))
+    [[ $RULE =~ D.*$BOOTSTRAP_IP.*ACCEPT ]] && iptables ${RULE}
+  fi
```

Comment:
https://stackoverflow.com/questions/67238417/how-to-pass-array-of-arguments-in-one-bash-script-to-another-where-some-argument

Reply:
To make the arguments passing safer.

Reply:
Done